### PR TITLE
fix: Show single emotes in the emote dropdown button

### PR DESCRIPTION
**Pull request by cyaiox**
This PR allows showing single emotes in the play emotes dropdown button.

Emotes in the Items tab:
image

Emotes in the Collection tab:
image

### CODE REVIEW

**Review by juanmahidalgo — CHANGES_REQUESTED** [review]
Testing the request change! Ignore this comment.

**Review by juanmahidalgo — CHANGES_REQUESTED** [review]
again, just trying to get this notified in Slack, ignore this comment 😂

**Review by juanmahidalgo — APPROVED** [review]

**Review by juanmahidalgo — COMMENTED** [review]
a comment without approval

**Review by juanmahidalgo — CHANGES_REQUESTED** [review]
a change request test

**Review by juanmahidalgo — APPROVED** [review]
ok, enough tests, approved for real now 😂  @cyaiox merge when ready

File: src/modules/item/sagas.ts [hunk 1]
```diff
@@ -360,10 +360,9 @@ export function* itemSaga(legacyBuilder: LegacyBuilderAPI, builder: BuilderClien
     const isEmotesFeatureFlagOn: boolean = yield select(getIsEmotesFlowEnabled)
     const { item } = action.payload
     const collectionId = item.collectionId!
-    if (openModals['EditItemURNModal']) {
-      yield put(closeModal('EditItemURNModal'))
-    } else if (openModals['EditPriceAndBeneficiaryModal']) {
-      yield put(closeModal('EditPriceAndBeneficiaryModal'))
+    const ItemModals = ['EditItemURNModal', 'EditPriceAndBeneficiaryModal', 'AddExistingItemModal']
```
Comment (cyaiox) [inline src/modules/item/sagas.ts:363]: This change is not related to this issue. It closes all opened modals when creating/updating an item.

File: src/components/CollectionProvider/CollectionProvider.tsx [hunk 2]
```diff
@@ -18,14 +22,41 @@ export default class CollectionProvider extends React.PureComponent<Props> {
...
+    if (
+      collection &&
+      paginatedItems &&
+      itemSelected &&
+      (this.state.initialPage === DEFAULT_ITEMS_PAGE || prevState.initialPage < this.state.initialPage)
+    ) {
+      if (!paginatedItems.find(item => item.id === itemSelected)) {
+        const totalPages = Math.ceil(items.length / itemsPageSize!)
+        const nextPage = Math.min(totalPages, (itemsPage as number[])!.length + 1)
+        if (!(itemsPage as number[]).includes(nextPage)) {
```
Comment (juanmahidalgo) [inline src/components/CollectionProvider/CollectionProvider.tsx:54]:
```suggestion
        if (Array.isArray(itemsPage) && !itemsPage.includes(nextPage)) {
```
:)

**Pull request merged**